src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php

```diff
@@ -24,9 +26,22 @@
  */
 class ValidatorExtension extends AbstractExtension
 {
+    /**
+     * @var ValidatorInterface
+     */
     private $validator;
 
-    public function __construct(ValidatorInterface $validator)
+    /**
+     * @var FormRendererInterface
+     */
+    private $formRenderer;
+
+    /**
+     * @var TranslatorInterface
```

Comment:
Please remove all added annotation on properties, the constructor is enough to define them.